installation/resources/installer-local.yaml

```diff
@@ -63,6 +63,149 @@ spec:
 ---
 apiVersion: v1
 kind: ServiceAccount
+metadata:
+  name: helm-certs-sa
+  namespace: kyma-installer
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: helm-certs-rolebinding
+  namespace: kube-system
+subjects:
+- kind: ServiceAccount
+  name: helm-certs-sa
+  namespace: kyma-installer
+roleRef:
+  kind: Role
+  name: helm-certs-installer
+  apiGroup: rbac.authorization.k8s.io
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: helm-certs-rolebinding
+  namespace: kyma-installer
+subjects:
+- kind: ServiceAccount
+  name: helm-certs-sa
+  namespace: kyma-installer
+roleRef:
+  kind: Role
+  name: helm-certs-installer
+  apiGroup: rbac.authorization.k8s.io
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: helm-certs-installer
+  namespace: kube-system
+rules:
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["get", "list"]
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: helm-certs-installer
+  namespace: kyma-installer
+rules:
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["get", "list", "create", "patch"]
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: helm-certs-job
+  namespace: kyma-installer
+  labels:
+    kyma-project.io/installation: ""
+spec:
+  template:
+    metadata:
+      name: helm-certs-job
+      namespace: kyma-installer
+      annotations:
+        sidecar.istio.io/inject: “false”
+    spec:
+      serviceAccountName: helm-certs-sa
+      restartPolicy: OnFailure
+      containers:
+      - name: certhelper
+        image: demonsthere/alpine-kubectl:latest
+        command:
```

Comment:
We should not use private mages and repos in our charts, is it possible to use xip image currently? If not then we need a new component with  pipeline

Reply:
Yes, this is just a temporary solution. I am looking into making this image as a bootstrap image in test-infra

Reply:
A new image has been introduced